src/components/navigation/Footer.tsx

```diff
@@ -110,16 +112,14 @@ const TopButton = styled.div`
 `
 
 const InfoContainer = styled.div`
-  flex-basis: 33.3333%;
-  display: flex;
-  flex-direction: column;
+  flex-shrink: 2;
+  width: 100%;
 `
 
 const Summary = styled.div`
   background-color: ${props => props.theme.colors.lightblue};
-  display: flex;
-  flex-direction: column;
-  padding: 12px 15px 0 15px;
+  padding: 32px 0 16px;
+  ${makeResponsivePadding}
```

Comment:
use explicit padding-top and padding-bottom here to make clear that the css helper is doing something